src/nsrt_learning/sampler_learning.py

```diff
@@ -281,6 +281,14 @@ def sampler(self, state: State, goal: Set[GroundAtom],
             x_lst.extend(state[goal_obj])  # add goal state
         x = np.array(x_lst)
         num_rejections = 0
+        if CFG.sampler_disable_classifier:
+            params = np.array(self._regressor.predict_sample(x, rng),
+                              dtype=self._param_option.params_space.dtype)
+            low = self._param_option.params_space.low
+            high = self._param_option.params_space.high
+            params = np.clip(params, low, high)
+            assert self._param_option.params_space.contains(params)
```

Comment:
You don't need to assert this, it's asserted in ParameterizedOption.ground in structs.py